scss/_variables.scss

```diff
@@ -1406,6 +1426,9 @@ $zindex-dropdown:                   1000 !default;
 $zindex-sticky:                     1020 !default;
 $zindex-fixed:                      1030 !default;
 $zindex-back-to-top:                1035 !default; // OUDS mod
+// fusv-disable
+$zindex-spinner:                    1038 !default; // OUDS mod
```

Comment:
Let's drop it, as it's not useful for now. And it's not a token that we will receive from Figma.